compendium/LogServiceImpl/test/TestLogService.cpp

```diff
@@ -163,13 +163,6 @@ TEST_F(LogServiceImplTests, InvalidLoggerUsage)
 {
     auto logger = GetLogger();
 
-    ASSERT_NO_THROW(logger->Log(static_cast<ls::SeverityLevel>(-1), "Test invalid negative severity level"));
-    EXPECT_FALSE(ContainsRegex(log_preamble + "Test invalid negative severity level"));
-
-    ASSERT_NO_THROW(logger->Log(static_cast<ls::SeverityLevel>(std::numeric_limits<unsigned int>::max()),
-                                "Test invalid maximum severity level"));
-    EXPECT_FALSE(ContainsRegex(log_preamble + "Test invalid maximum severity level"));
-
```

Comment:
what was the reason for removing these test points

Reply:
These are removed since earlier there was no default level for logging and these values of log level were considered invalid. With new OSGi spec, the default logging level is 'trace' and hence the messages are logged at that level.

Reply:
Modified test instead of removing. Thanks.